### PR TITLE
add disclaimer/public_note to 6 MO counties

### PR DESCRIPTION
**Pull request by TomGoBravo**
This PR adds a disclaimer/public_note to 6 MO counties following up on https://trello.com/c/6roEMYkZ/1020-switch-to-usa-facts-for-6-mo-counties-nyt-is-under-reporting-kc-joplin-data

The note is added to `cases`. It is not added to `new_cases` because `new_cases_and_deaths` runs after `manual_filter` and doesn't copy tags. It isn't added to `deaths` or `new_deaths` because they aren't in `_METRIC_TO_FIELDS`.

## Tested

`./run.py data update` and `git difftool --no-prompt -t vimdiff main -- data/multiregion-wide-dates.csv` shows only changes described above.

### CODE REVIEW

**Comment by TomGoBravo** [comment]
[Switch to USA Facts for 6 MO counties (NYT is under reporting KC/Joplin data)](https://trello.com/c/6roEMYkZ/1020-switch-to-usa-facts-for-6-mo-counties-nyt-is-under-reporting-kc-joplin-data)

File: data/region-overrides.json [hunk 1]
```diff
@@ -132,6 +132,22 @@
       "region": "48085",
       "context": "https://trello.com/c/WCCU7ey1/1417-blocked-data-after-5-27-cases-in-collin-county-tx-stalled"
     },
+    {
+      "include": "region",
+      "metric": "metrics.caseDensity",
+      "blocked": false,
+      "region": "29037, 29047, 29095, 29165",
+      "context": "https://trello.com/c/6roEMYkZ/1020-switch-to-usa-facts-for-6-mo-counties-nyt-is-under-reporting-kc-joplin-data",
+      "disclaimer": "Four counties (Cass, Clay, Jackson and Platte) overlap the municipality of Kansas City, MO. Our data source has all cases and deaths for the municipality in Jackson County."
```
Comment (mikelehen) [inline data/region-overrides.json:141]: Maybe change the second sentence to:

> Our data source attributes all cases and deaths for Kansas City to Jackson County.

File: data/region-overrides.json [hunk 2]
```diff
@@ -132,6 +132,22 @@
...
+    },
+    {
+      "include": "region",
+      "metric": "metrics.caseDensity",
+      "blocked": false,
+      "region": "29097, 29145",
+      "context": "https://trello.com/c/6roEMYkZ/1020-switch-to-usa-facts-for-6-mo-counties-nyt-is-under-reporting-kc-joplin-data",
+      "disclaimer": "Two counties (Jasper and Newton) overlap the municipality of Joplin, MO. Our data source has all cases and deaths for the municipality in Jasper County."
```
Comment (mikelehen) [inline data/region-overrides.json:149]: Same.
> Our data source attributes all cases and deaths for Joplin to Jasper County.

**Review by mikelehen — APPROVED** [review]
Thanks! Minor wording change suggestion, but you can ship as-is if you're not a fan.

**Pull request merged**